src/mmw/js/src/analyze/views.js

```diff
@@ -57,6 +57,33 @@ var ResultsView = Marionette.LayoutView.extend({
             newProjectUrl = '/project/new/' + modelPackageName,
             projectUrl = '/project';
 
+        if (modelPackageName === 'gwlfe') {
```

Comment:
A 1 sq km AOI in the middle of Delaware Bay returns this:

<img width="1275" alt="screen shot 2016-09-28 at 3 43 13 pm" src="https://cloud.githubusercontent.com/assets/4165523/18929400/51b10e3a-8592-11e6-94d7-d63354375237.png">

&

<img width="1268" alt="screen shot 2016-09-28 at 3 43 38 pm" src="https://cloud.githubusercontent.com/assets/4165523/18929411/5ced2388-8592-11e6-9ab1-7dd8eeeb1ac9.png">